tools/decompress_baserom.py

```diff
@@ -129,23 +130,14 @@ def per_version_fixes(file_content: bytearray, version: str) -> bytearray:
     if version == "gc-eu-mq-dbg":
         # Strip the overdump
         print("Stripping overdump...")
```

Comment:
```suggestion
         # Strip the overdump
         # That is, an area filled with 0xFF that may be from erased EEPROM data, followed by rom data from another unrelated game
         print("Stripping overdump...")
```
Or something (may need a line break too)